### ISSUE TITLE
Rotational to kinetic energy dupe bug

### ISSUE THREAD

**Issue by tyra314** [malte0811/IndustrialWires#22]
When I'm connecting a rotational-to-kinetic converter to an IC2 kinetic generator, I'm getting 37,5 EU/t even without inputting anything into the converter.

![Example setup](https://user-images.githubusercontent.com/16997177/34468496-4a317658-ef0a-11e7-8edd-bc164408db33.png)

I'm using:
- Minecraft: 1.12.2
- Forge: 14.23.1.2582
- Industrial Wires: 1.6-23
- IE: 0.12-76
- IC2: 2.8.35-ex112

**Comment by malte0811** [comment]
This is probably an IC2 bug, unless the `IKineticSource` interface was changed between 2.8.27 (doc version I have here) and 2.8.35 (build I used for testing). The return value of `requestkineticenergy` seems to be ignored (it is called though), the kinetic generator seems to just use `maxrequestkineticenergyTick`. The documentation (on build 27) explicitely states that this method should return the theoretical bandwidth,  not the available amount. To me that means returning e.g. 1000 in `maxrequest...` and returning 0 in `request...` should not produce any energy. @sfPlayer1
